agent/tool-scripts/dcgm

```diff
@@ -1,6 +1,13 @@
 #!/usr/bin/python3
 # -*- mode: python -*-
 
+"""
+This script was originally created to fit dcgm into the transient tool model
```

Comment:
Can we consider re-writing this message to talk about the present and exclude the past?  Something like:

```
"""Place-holder "dcgm" persistent tool, providing a help message like the
other existing transient tools, and issuing a warning indicating this script
is not actually executed directly.
"""
```